src/accordion/accordion.ts

```diff
@@ -199,6 +207,14 @@ export class NgbAccordion implements AfterContentChecked {
    */
   isOpen(panelId: string): boolean { return this._states.get(panelId); }
 
+  /**
+   * @internal
```

Comment:
Using `@internal` will make it fail in the AoT mode, please remove. I think that it would be better to just in-line this logic in a template and IMO we should just remove this method altogether.